ResuMe/urls.py

```diff
@@ -15,9 +15,9 @@
 """
 from django.contrib import admin
 from django.urls import path
-from resumeApp import views
+from Feed import views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.resumeApp, name='resumeApp')
+    path('', views.Feed, name='Feed')
```

Comment:
According to Nati's review, change this line to `path('', views.feed, name='Feed')` as well

Reply:
Ok